.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+1be3f1bfb671fa6d59dcd2203f23e1625194574b
```

Comment:
Uhm. Not a big fan, but also don't care much about these. So... not gonna fight it too much ;)

Reply:
I don't particularly like it but it does give us a good way of configuring our git clients to have a sane `git blame` output.